### DISCUSSION TITLE
FM Radio commands

### DISCUSSION THREAD

**Discussion by z34pec** [egzumer/uv-k5-firmware-custom#98] [Q&A] [answered]
Hello to everyone.

What are commands in FM radio mode ?

I just know that pressing M is for adding stations to a memory, long press 1 is for auto scanning and adding stations to memory, long press 3 is for scanning but that is all (is it also correct as i said ?) .

Any other commands ?

**Comment by ErikS-web** [comment, marked as answer]
Just the original. See manual page 15.
[UV-K5(8)_K5-3_ENG_User`s_Manual.pdf](https://github.com/egzumer/uv-k5-firmware-custom/files/13305366/UV-K5.8._K5-3_ENG_User.s_Manual.pdf)
